src/Backend/Modules/Blog/Layout/Templates/Edit.html.twig

```diff
@@ -146,7 +158,7 @@
                       <div class="form-group last">
                         <label for="tags" class="control-label">{{ 'lbl.Tags'|trans|ucfirst }}</label>
                         {% form_field tags %} {% form_field_error tags %}
-                        <div class="help-block">{{ 'msg.HelpTextTags'|trans }}</div>
+                        <div class="help-block" id="tags-info">{{ 'msg.HelpTextTags'|trans }}</div>
```

Comment:
why did you add the id?

Reply:
give me some time, something goes wrong :(